mchf-eclipse/drivers/ui/menu/ui_menu.c

```diff
@@ -626,7 +626,7 @@ const char* UiMenu_GetSystemInfo(uint32_t* m_clr_ptr, int info_item)
       		snprintf(out,32, "D%s", UHSDR_VERSION+4);
 		  #endif
 		#else
-    	  snprintf(out,32, "%s", UHSDR_VERSION+4);
+			snprintf(out,32, "X%s", UHSDR_VERSION+4); // eXperimental fw build
     	#endif
```

Comment:
Why do you think every build which is not made by me is "experimental"? I do not think so... They are not official builds so trailing "D" misses. But no other trail is needed.... Pse remove this.

Reply:
seemed like a good idea at the time. reverted. gain tweaked. more gain was not better especially when CMP is enabled